src/wrapped_radix_engine_toolkit/high_level/manifest_building/manifest_assets_transfers.rs

```diff
@@ -62,12 +62,7 @@ impl TransactionManifest {
             }
         }
 
-        let scrypto_manifest = builder.build();
-
-        TransactionManifest::from_scrypto(
-            scrypto_manifest,
-            from_account.network_id(),
-        )
+        TransactionManifest::sargon_built(builder, from_account.network_id())
```

Comment:
this unwraps the failable constructor - which is OK, since this manifest is built by US in Sargon, and we know the SBOR depth.